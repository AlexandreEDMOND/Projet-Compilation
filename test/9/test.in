int main()
{
  float a = -6.5;
  float e = 1856.75982;

  int c = -86;
  int d = 69;

  a++;
  e--;
  c--;
  d++;

  print(a);
  print(e);
  print(c);
  print(d);
  
  return 0;
}