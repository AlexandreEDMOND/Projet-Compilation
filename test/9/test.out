-5.50000000
1855.75976562
-87
70
